### PR TITLE
Scatterplot controls

### PR DESCRIPTION
**Pull request by aronhelser**
fix(ScatterPlot): use scatterplot controls and manager
fix(ScatterPlot): handle color maps and changing color-by array

### CODE REVIEW

**Comment by jourdain** [comment]
can you build the dist in release mode and add it to that PR?

File: Server/scatterplotProtocol.py [hunk 1]
```diff
@@ -16,26 +16,104 @@
 # data, math
 import vtk, vtk.util.numpy_support
 # from vtk.util.numpy_support import vtk_to_numpy, numpy_to_vtk
-# from vtkPVServerManagerRenderingPython import vtkSMPVRepresentationProxy
+from vtkPVServerManagerRenderingPython import vtkSMPVRepresentationProxy
```
Comment (jourdain) [inline Server/scatterplotProtocol.py]: Do you really need that? It fails on my mac and not having it just works.
Maybe we want to have it in a try/catch if it is really needed

Comment (aronhelser) [reply Server/scatterplotProtocol.py]: `vtkSMPVRepresentationProxy` is used to update the color maps. That's why you are seeing white points, probably. Why would it fail? You used to run on your mac with this line, right?

Comment (jourdain) [reply Server/scatterplotProtocol.py]: I don't know. The script crash at the import for me.

Comment (jourdain) [reply Server/scatterplotProtocol.py]: it doesn't find vtkPVServerManagerRenderingPython

**Pull request merged**